hpOneView/storage.py

```diff
@@ -99,53 +252,308 @@ def add_storage_pool(self, name, storageSystemUri, blocking=True,
                 return server
         return task
 
-    # Temporarly modify the headers passed for POST and DELTE on storage volume
-    # templates in order to work around a bug. Without these headers the call
-    # cause a NullPointerException on the appliance and a 400 gets returned.
-    def add_storage_volume_template(self, name, capacity, shareable, storagePoolUri, state='Normal',
-                                    description='', provisionType='Thin', verbose=False):
+    def get_storage_pool_by_id(self, sto_pool_id):
+        """Gets the storage pool resource for the specified ID.
+
+        :param sto_pool_id: ID of specific storage pool
+        """
+        url = make_url(uri['storage-pools'], sto_pool_id)
+
+        body = self._con.get(url)
+        return body
+
+    def remove_storage_pool(self, sto_pool_id, force=False, blocking=True, verbose=False):
+        """Removes an imported storage pool from OneView
+
+        :param sto_pool_id: ID of specific storage pool that will be removed
+        :param force: if set to true the operation completes despite any problems with network connectivity or errors on
+        the resource itself. The default is false
+
+        :param blocking:
+        :param verbose
+        """
+        url = make_url(uri['storage-pools'], sto_pool_id, force=force)
+
+        task, body = self._con.delete(url)
+        if blocking is True:
+            task = self._activity.wait4task(task, tout=600, verbose=verbose)
+            return body
+        return task
+
+    ###########################################################################
+    # Storage Volume Attachments: resource represents the connection information for the volume.
+    ###########################################################################
+
+    def get_attachments_volumes(self, filter=''):
```

Comment:
Maybe rename to get_volume_attachments.  For the functions below related to attachments I would use _volume_attachments in place of _attachments_volumes